source/elements/oneDAL/source/common_interface/error_handling.rst

```diff
@@ -14,8 +14,8 @@ Exception classification
 Exception classification in oneDAL is aligned with C++ Standard Library
 classification. oneDAL shall introduce abstract classes that define the base
 class in the hierarchy of exception classes. Concrete exception classes are
-derived from respective C++ Standard Library exception classes. oneDAL library
-shall throw exceptions represented with concrete classes.
+derived from the respective C++ Standard Library exception classes. oneDAL
+library shall throw exceptions represented with concrete classes.
```

Comment:
oneDAL already has "library" in its name, so remove "library" here

Reply:
fixed